### PR TITLE
zdump: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
The man page has links to tzfile, zic, tzset, tzselect. These could be worth of a lets document issue.

File: pages/linux/zdump.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# zdump
+
+> Print timezone information.
+> More information: <https://manned.org/zdump.8>.
+
+- Print the current time in a zone:
+
+`zdump {{zone}}`
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/zdump.md:8]: The `zone` can be for example `Japan` or `America/New_York`. In my mind "timezone" refers to the whole strip that covers a specific time whereas "Japan" is just a zone within a timezone.

[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/zdump.md:8]: Alright, that's fair.

File: pages/linux/zdump.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# zdump
+
+> Print timezone information.
+> More information: <https://manned.org/zdump.8>.
```
Comment (spageektti) [inline pages/linux/zdump.md:4]:
```suggestion
> More information: <https://manned.org/zdump>.
```
Thats the same page.

File: pages/linux/zdump.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+- Print the current time in a timezone:
+
+`zdump {{timezone}}`
```
Comment (spageektti) [inline pages/linux/zdump.md:8]:
```suggestion
`zdump {{timezone}}`

- Display help:

`zdump --help`
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**